onset_maproom/layout.py

```diff
@@ -169,19 +169,32 @@ def controls_layout():
             ),
             html.P(
                 """
-                The Maproom explores historical rainy season onset and
-                cessation dates based on user-defined definitions. The
-                date when the rainy season starts is critical to
-                agriculture planification, in particular for planting.
+                The Maproom explores current and historical rainy season onset
+                """+CONFIG["and_cess_text"]+"""
+                 dates based on user-defined definitions.
+                The date when the rainy season starts with germinating rains
+                is critical to agriculture planification, in particular for planting.
                 """
             ),
             html.P(
                 """
-                By enabling the exploration of the history of onset
-                dates, the Maproom allows to understand the spatial
-                and temporal variability of this phenomenon and
-                therefore characterize the risk for a successful
-                agricultural campaign associated with it.
+                The default map shows whether germinating rains have occured
```

Comment:
In my experience, there are choices to make to accommodate first time users of the page, and returning ones. There is a conflict because first-time users need all the info about the app, whereas returning users want to easily jump to the controls (the already know what all is about). In light of this, my practice has been:

1. first thing first: describe what is seen when landing (maybe should us "map of landing page" rather "default map")
2. controls
3. detailed description of the choices and contents

Putting more of 3. in 1. is annoying for returning users by pushing back the controls. Having less of 1. is not ideal because bad users could then jump to controls, never read anything and then ask annoying questions.

But I am totally open to new strategy, possibly enabled by new techno we didn't use in Ingrid Maprooms. Like collapsable text for instance?

Reply:
> first thing first: describe what is seen when landing (maybe should us "map of landing page" rather "default map")

What about "first thing first: describe what is currently displayed?" The explanatory text can change depending on the value of the control.

Reply:
That sounds like a new strategy enabled by new techno as we were not (much) doing dynamic text in Ingrid Maprooms. Probably need signaling that text is updating, maybe some blinking or what-have-you for the user to realize that it is.
Will make an issue out of this as I don't think I have time to do that on this PR.